File: g8/Clasificador.java
public class Clasificador
{
    public static void main(String args[]){
        int ladoA = Lee.leeInt("Ponga el lado A");
        int ladoB = Lee.leeInt("Ponga el lado B");
        int ladoC = Lee.leeInt("Ponga el lado C");
        if(ladoA+ladoB>ladoC && ladoB+ladoC>ladoA && ladoA+ladoC>ladoB){
            if(ladoA == ladoB && ladoA == ladoC && ladoB == ladoC){
                System.out.println("El triangulo es equilatero");
            }else{
                //tenemos dos alternativas que seria
                //triangulo Escaleno e Isoceles
                if(ladoA != ladoB && ladoA != ladoC && ladoB != ladoC){
                    System.out.println("El triangulo es Escaleno");
                }else{
                    System.out.println("El triangulo es Isoceles");
                }
            }
        }else{
            System.out.println("No es un triangulo");
        }
        }
}

File: g10/Clasificador.py
def clasificar_triangulo(ladoA: int, ladoB: int, ladoC: int) -> str:
    respuesta = "Es un triangulo valido"

    if ladoA + ladoB > ladoC and ladoB + ladoC > ladoA and ladoA + ladoC > ladoB:
        if ladoA == ladoB and ladoA == ladoC and ladoB == ladoC:
            respuesta = "El triangulo es equilatero"
        elif ladoA != ladoB and ladoA != ladoC and ladoB != ladoC:
            respuesta = "El triangulo es Escaleno"
        else:
            respuesta = "El triangulo es Isoceles"
    else:
        respuesta = "No es un triangulo"

    return respuesta